spec/controllers/reviews_controller_spec.rb

```diff
@@ -136,6 +136,40 @@ def new_reviewer
       path = session_review_path(conference, assigns(:session), assigns(:review))
       expect(response).to redirect_to(path)
     end
+
+    context 'create action should create a third reject' do
+      let(:reject_recommendation) { Recommendation.create(name: 'strong_reject') }
+      let(:approve_recommendation) { Recommendation.create(name: 'strong_accept') }
+      before { Rating.create(title: 'rating.low.title') }
+
+      it 'when there are other two other rejections' do
+        session.final_reviews.new(recommendation: reject_recommendation).save(validate: false)
+
+        post :create, session_id: session.id, final_review: valid_final_review_params
+
+        session.reload
+
+        expect(session).to have(3).final_reviews
+        expect(session.final_reviews.map(&:recommendation)).to all(be_strong_reject)
+      end
+
+      it 'not when this is the first review' do
+        post :create, session_id: session.id, final_review: valid_final_review_params
+
+        session.reload
+
+        expect(session).to have(1).final_reviews
+      end
+
+      it 'not when this is an approve review and there are one rejection' do
```

Comment:
How about when there is already one approved review, and we're adding a new reject one?
And how about when there is a approved and a reject reviews, and we're adding a third that is an rejection?

What happens with the sessions that already have two rejections? Is there a strategy for that?